docs/customization/menu.rst

```diff
@@ -41,32 +40,84 @@ In the example below we are adding a one new item to the Admin panel menu and a
         {
             $menu = $event->getMenu();
 
-            $menu->addChild('backend_main')
-                ->setLabel('Test Backend Main');
+            $newSubmenu = $menu->addChild('new')
+                ->setLabel('New Admin Panel Menu');
+
+            $newSubmenu->addChild('new-subitem')
+                ->setLabel('New Item');
         }
+    }
+
+**2.** After creating your class with a proper method for the menu customizations you need, subscribe your
+listener to the ``sylius.menu.admin.main`` event in the ``AppBundle/Resources/config/services.yml``.
```

Comment:
Do we do that for any other of the services? I mean, we do not override a service nor a parameter. We declare a new one, that lives inside the AppBundle.